internal/downlink/ack/ack_test.go

```diff
@@ -121,6 +121,7 @@ func TestDownlinkAck(t *testing.T) {
 			HandleTxAckRequest: &as.HandleTxAckRequest{
 				DevEui: []byte{1, 2, 3, 4, 5, 6, 7, 8},
 				FCnt:   10,
+				TxInfo: &txInfo1,
```

Comment:
Isn't the `GatewayId` expected in `HandleTxAckRequest`?

Reply:
Added `GatewayId` ub test

Thanks